sunspec/core/modbus/mbmap.py

```diff
@@ -22,12 +22,17 @@
 """
 
 import struct
+import sys
 
 try:
     import xml.etree.ElementTree as ET
 except:
     import elementtree.ElementTree as ET
 
+# Python 3 compatibility for long()
+if sys.version_info > (3,):
+    long = int
```

Comment:
My experience is in python 3 rather than python 2 and so I was not sure. One reason to keep the calls to `long()` though would be that it would make a long type even if the number is not large enough to convert and int to long. This would be necessary for the struct pack commands to have the correct size types and is why I used the `if` statement rather than getting rid of long.